Tasks/DockerV2/dockerpush.ts

```diff
@@ -79,3 +99,58 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
 
     return promise;
 }
+
+async function publishToImageMetadataStore(connection: ContainerConnection, imageName: string, tags: string[], digest: string, dockerFilePath: string): Promise<any> {
+    const imageUri: string = getResourceName(imageName, digest);
+    const dockerFileContent: string = fs.readFileSync(dockerFilePath, 'utf-8').toString();
+    const baseImageName = getBaseImageName(dockerFileContent);
+    const layers = dockerCommandUtils.getLayers(connection, imageUri);
```

Comment:
Have separate block for getting pipeline variables and separate block for getting layers/base image name/imageUri

Reply:
Done